### DISCUSSION TITLE
Surface Integral Setup

### DISCUSSION THREAD

**Discussion by TLWise** [idaholab/moose#25014] [Q&A Modules: General]
I would like to determine the drag and lift force on an object in fluid flow using the surface integral involving the viscosity multiplied by the normal derivative of the fluid velocity summed with the surface integral of the normal pressure. Is there an example of setting up a post processor to accomplish this?

**Comment by GiudGiud** [comment]
@grmnptr just made a pull request to add this for finite volume. It should be in the framework soon. 

Do you need this for FE or FV fluid flow?

**Reply by GiudGiud** [reply]
#25013

**Reply by TLWise** [reply]
Thanks for the response. I would like to implement this in FE for fluid flow.

**Reply by GiudGiud** [reply]
It should be very similar for FE we just need to add the computeQpIntegral implementation on those postprocessors
